src/select/types.js

```diff
@@ -22,6 +22,8 @@ export type OptionT = $ReadOnly<{
   disabled?: boolean,
   clearableValue?: boolean,
   isCreatable?: boolean,
+  // eslint-disable-next-line flowtype/no-weak-types
+  [key: string]: any,
```

Comment:
does replacing `any` with `mixed` essentially put it back to before this change? i'm a bit hesitant to throw out all of the type safety on this option.

Reply:
I think `mixed` will work. My understanding is that `any` allows for dynamic typing of values which I don't believe would be necessary here 👍